src/Manager.php

```diff
@@ -202,10 +207,9 @@ public function addRole(Role $role): self
         return $this;
     }
 
-    public function removeRole(string $name): self
+    public function getRole(string $name): ?Role
     {
-        $this->removeItem($name);
-        return $this;
+        return $this->itemsStorage->getRole($name);
```

Comment:
This method may returns permission instead of role. This will result in a fatal error.